README.md

```diff
@@ -2,12 +2,37 @@
 
-| Package           | Type        | Version                                                                                                                                                   |
-|-------------------|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
-| core              | library     | [![NPM Version](http://img.shields.io/npm/v/@foundry-cli/core.svg?style=flat)](https://www.npmjs.org/package/@foundry-cli/core)                           |
-| prompts           | application | [![NPM Version](http://img.shields.io/npm/v/@foundry-cli/prompts.svg?style=flat)](https://www.npmjs.org/package/@foundry-cli/prompts)                     |
-| generator-example | library     | [![NPM Version](http://img.shields.io/npm/v/@foundry-cli/generator-example.svg?style=flat)](https://www.npmjs.org/package/@foundry-cli/generator-example) |
+## Packages
 
-Can you champion this timeframe you gotta smoke test your hypothesis eat our own dog food. Collaboration through advanced technlogy.
+| Package         | Type        | Version                                                                                                                                               |
+| --------------- | ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| prompts         | library     | [![NPM Version](http://img.shields.io/npm/v/@foundry-cli/prompts.svg?style=flat)](https://www.npmjs.org/package/@foundry-cli/prompts)                 |
+| loader          | library     | [![NPM Version](http://img.shields.io/npm/v/@foundry-cli/loader.svg?style=flat)](https://www.npmjs.org/package/@foundry-cli/loader)                   |
+| generator       | library     | [![NPM Version](http://img.shields.io/npm/v/@foundry-cli/generator.svg?style=flat)](https://www.npmjs.org/package/@foundry-cli/generator)             |
+| create-workleap | application | [![NPM Version](http://img.shields.io/npm/v/@foundry-cli/create-workleap.svg?style=flat)](https://www.npmjs.org/package/@foundry-cli/create-workleap) |
 
-Put it on the parking lot this proposal is a win-win situation which will cause a stellar paradigm shift, and produce a multi-fold increase in deliverables. Finance on-brand but completeley fresh, or who's responsible for the ask for this request?, but let's put a pin in that, so critical mass. In an ideal world locked and loaded prairie dogging workflow ecosystem. Open door policy no scraps hit the floor. This is a no-brainer. Reach out face time.
\ No newline at end of file
+## Usage
+
+To create a new project base on the Workleap starter kit, run:
+
+### npm
+
+```cmd
+npm create @foundry-cli/workleap <your-app-name>
```

Comment:
are the orders still valid with the package `create-workleap` ?

Reply:
Ha no... I forgot to update that file.